contrib/ros/envs/sawyer_env.py

```diff
@@ -0,0 +1,356 @@
+"""
+ROS environment for the Sawyer robot
+Every specific sawyer task environment should inherit it.
+"""
+
+import copy
+import os.path as osp
+
+import numpy as np
+import rospy
+import rospkg
+from gazebo_msgs.msg import ModelStates
```

Comment:
nit: alphabetical order

Reply:
Do you mean the order of rospy and rospkg? I am not sure about how to sort import ... statement and from ... import ... statement.

Reply:
consider `from ... import ...` and `import ...` to be the same for alphabetical purposes, and sort alphabetically based on the first appearance of a package name.

e.g. this block would be sorted
```python
from gazebo_msgs.msg import ModelStates
import intera_interface
import numpy as np
import rospkg
import rospy
```
because `gazebo_msgs.msg` comes before `intera_interface` alphabetically regardless of the `from`, and because `rospkg` comes before `rospy`